lib/diagnostic.js

```diff
@@ -6,26 +6,46 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function () {
-
+const normalizeWords = function (str) {
+  let normalWords = [];
+  normalWords = str.split(/\s+/);
+  return normalWords;
 };
 
 // 2. Write a function that takes a string arguments
 // and returns an array of unique normalized words.
-const uniqueWords = function () {
-
+const uniqueWords = function (str) {
+  let uniqueKeys = [];
+  let normalWords = normalWords(str);
```

Comment:
your calling your variable `normalWords` here which this functions scope won't know about, plus it's a variable so you can't pass an argument to it. you should be calling `normalizeWords(str)` which is the function you defined above